src/components/ebay-checkbox/index.js

```diff
@@ -0,0 +1,30 @@
+const markoWidgets = require('marko-widgets');
+const emitAndFire = require('../../common/emit-and-fire');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const classes = ['checkbox', input.class];
```

Comment:
You can skip the intermediate var and declare this directly in the object below.